README.md

```diff
@@ -1 +1,122 @@
+See the [Status Sync Add-on README](addons/hub/status_sync/README.md) for more details.
+
+See the [Workflow example](example/hello-world.yaml) for the required label(s) and annotation(s).
+
+## Dependencies
+- The Open Cluster Management (OCM) multi-cluster environment needs to be setup. See the [OCM website](https://open-cluster-management.io/) on how to setup the environment.
+- In this multi-cluster model, OCM will provide the cluster inventory and ability to deliver workload to the remote/managed clusters.
+
+## Developer Build
+1. Build the project.
+```
+make docker-build
+```
+
+2. Optional: load the built image to KinD clusters.
+```
+kind load docker-image --name=hub quay.io/open-cluster-management/argo-workflow-multicluster:latest
+kind load docker-image --name=cluster1 quay.io/open-cluster-management/argo-workflow-multicluster:latest
+```
+
+
+## Getting Started
+1. Setup an OCM Hub cluster and registered at least one OCM Managed cluster.
+
+2. On the hub cluster, install the Argo Workflow and Argo Workflow Status Result CRDs:
+```
+kubectl apply -f hack/crds/
+```
+
+3. On the hub cluster, install the OCM Argo Workflow Install Addon by running:
+```
+kubectl apply -f deploy/addon/install/
+```
+This will automate the installation of Argo Workflow to the managed clusters. See the [Install Add-on README](addons/hub/install/README.md) for more details.
+For manual installation of Argo Workflow, elevate the OCM agent permission to access 
+the Workflow objects by `kubectl apply -f example/managed`.
+This manual privilege escalation is not necessary when using the OCM Argo Workflow Addon.
+
+4. On the hub cluster, install the OCM Argo Workflow Status Sync Addon by running:
+```
+kubectl apply -f deploy/addon/status-sync/
+```
+This will install the status sync agent to all the managed clusters. See the [Status Sync Add-on README](addons/hub/status_sync/README.md) for more details.
+
+
+5. On the hub cluster, deploy the Argo Workflow Multicluster manager:
+```
+kubectl apply -f deploy/argo-workflow-multicluster/
+```
+
+6. On the hub cluster, apply the ManagedClusterSetBinding and Placement.
+```
+kubectl apply -f example/clusterset-binding.yaml
+kubectl apply -f example/workflow-placement.yaml
+```
+
+7. On the hub cluster, create the example Workflow.
+```
+kubectl apply -f example/hello-world.yaml
+```
+
+8. On the managed cluster, check the Workflow that was executed.
+```
+$ kubectl get workflow
+NAME                       STATUS      AGE     MESSAGE
+hello-world-multicluster   Succeeded   3m52s
+```
+
+9. On the hub cluster, check the Workflow to see the status is now synced from the managed cluster.
+```
+$ kubectl get workflow hello-world-multicluster -o yaml 
+...
+status:
+...
+  conditions:
+  - status: "False"
+    type: PodRunning
+  - status: "True"
+    type: Completed
+  finishedAt: "2022-11-12T21:57:19Z"
+...
+  phase: Succeeded
+  progress: 1/1
+  resourcesDuration:
+    cpu: 9
+    memory: 9
+...
+```
```

Comment:
1. Line 40: "Setup" -> "Set up"
2. Line 40: "registered" -> "register"
3. Line 51 and 52: "Argo Workflows" -> "Argo Workflows"